src/engine/guidance/collapse_scenario_detection.cpp

```diff
@@ -123,7 +123,7 @@ bool isStaggeredIntersection(const RouteStepIterator step_prior_to_intersection,
 
     const auto angle = [](const RouteStep &step) {
         const auto &intersection = step.intersections.front();
-        const auto entry_bearing = intersection.bearings[intersection.in];
+        const auto entry_bearing = util::bearing::reverse(intersection.bearings[intersection.in]);
```

Comment:
From the values @vng posted, the change should be fine. It is void, since `is_right` and `is_left` are used in combination (right + left), but it might prevent future bugs.